config.toml

```diff
@@ -18,9 +18,10 @@ theme = "hugo-elate-theme"
     brand = 'JavaFX'
     home = "Home"
     documentation = "Documentation"
-    # events = "Upcoming Events"
+    events = "Upcoming Events"
     work = "Community"
     testimonials = "Testimonials"
+    videos = "Videos"
```

Comment:
Please create an issue for this, so we can discuss if the site should contain this new section.